pages/guides/transfer-between-users/create-verified-funding-source.mdx

```diff
@@ -99,13 +108,15 @@ Joe Buyer will complete the IAV flow by authenticating with his online banking c
 
 ##### Sample Response
 
-```Rawnoselect
+```javascript
```

Comment:
If we want to we could add the [`json` highlighter](https://github.com/highlightjs/highlight.js/tree/master/src/languages) for examples like this so it will say "JSON" instead of "JavaScript" here:

![image](https://user-images.githubusercontent.com/309638/82057609-3b6e6d00-9689-11ea-88de-800f69fa2579.png)

Reply:
yeah, realized that after I went through and updated these. Thought you had mentioned that they should be tagged as javascript, but realized after adding html to highlight.js that json was an acceptable type.